src/AWS.Deploy.Common/Recipes/OptionSettingItem.ValueOverride.cs

```diff
@@ -150,7 +152,7 @@ public void SetValueOverride(object valueOverride)
                 {
                     if (deserialized.TryGetValue(childOptionSetting.Id, out var childValueOverride))
                     {
-                        childOptionSetting.SetValueOverride(childValueOverride);
+                        childOptionSetting.SetValueOverride(childValueOverride, recommendation);
                     }
                 }
             }
```

Comment:
I believe this was supplanted by `OptionSettingHandler` via https://github.com/aws/aws-dotnet-deploy/pull/548, but let me know otherwise. 

I still needed to pass `recommendation` into all of the `Validate` calls, which we discussed at the 5/5/22 internal meeting.